api/types/client.go

```diff
@@ -292,6 +292,7 @@ type ServiceListOptions struct {
 // TaskListOptions holds parameters to list  tasks with.
 type TaskListOptions struct {
 	Filter filters.Args
+	All    bool
```

Comment:
I was following the same pattern in `docker ps` to keep it consistent. I agree though 👍